javageci-tools/src/test/java/javax0/geci/lexeger/TestJavaLexed.java

```diff
@@ -17,6 +18,15 @@ void testNullTransformation(List<String> lines) {
         Assertions.assertEquals(expected, result);
     }
 
+    private String buildLexemeTree(JavaLexed sut) {
```

Comment:
`buildLexemeTree()` is not a good name. It is not a tree. It is a sequential string representation of the list of lexical elements. I would use the method name `toString(JavaLexed javaLexed)`

Reply:
Renamed to toLexicalString.